dynawo/sources/Modeler/Common/DYNCompiler.h

```diff
@@ -66,13 +66,15 @@ class Compiler {
    * @param modelicaModelsDirs modelica models directories
    * @param modelicaModelsExtension: modelica models extension
    * @param additionalHeaderFiles: list of headers that should be included in the dynamic model files
+   * @param rmModels: remove .mo model (defautl: false)
```

Comment:
the default might be anything depending on the compiler currently, please remove the parenthesis or define a default explicitly